cephfs/path_test.go

```diff
@@ -222,6 +222,7 @@ func TestSymlink(t *testing.T) {
 	})
 
 	t.Run("symlinkExistsError", func(t *testing.T) {
+		useMount(t)
```

Comment:
I'm confused as to why this sub-test needs useMount. Maybe I'm overlooking it but I don't see any use of an OS mountpoint for this. Can you point out what I'm missing?

Reply:
It depends on the test before, which depends on the mountpoint.

Reply:
Ooh. Gotcha. We shouldn't have it depend on the other subtest, but that's a fix for another day.